config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3974,6 +3974,13 @@
 # ########################################################################
 # Notification Preferences
 # ########################################################################
+
+# Modify the default option users will have for Announcement notifications (User Preferences).
+# 1 = Do not send me low priority announcements
+# 2 = Send me one email per day summarizing all low priority announcements
+# 3 = Send me each notification separately [DEFAULT]
+# prefs.default.noti.option.announcements=3
```

Comment:
Maybe prefs.notification.announcements.default.option ?

I mean, not the best person to define it, but....
prefs -> Preferences section
notification -> Notificaton section
announcements -> tool section
default -> well, it's the default option

WDYT?

Reply:
I though about a similar order. But finally set the tool name at the end just to make it easier to read in the future, if this kind of property is developed for the rest of tools

Example:
prefs.default.noti.option.announcements
prefs.default.noti.option.samigo
prefs.default.noti.option.assignments

Still prefering `prefs.notification.announcements.default.option` ?